scripts/xtensa-build-zephyr.py

```diff
@@ -902,6 +908,73 @@ def build_platforms():
 			symlinks=True, ignore_dangling_symlinks=True, dirs_exist_ok=True)
 
 
+def install_lib(platform, sof_output_dir, abs_build_dir):
+	"""[summary] Sign loadable llext modules, if any, copy them to the
+	deployment tree and create UUID links for the kernel to find and load
+	them."""
+
+	global signing_key
+	global platform_wconfig
+
+	with os.scandir(str(abs_build_dir)) as iter:
```

Comment:
Is it really required to scan the ENTIRE build directory?

Reply:
@marc-hb this isn't recursive, is it? If not - shouldn't be *that* bad? (I just tested - it isn't) What would be a better way?

Reply:
My bad I confused .scandir() with .walk().